docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -61,10 +61,75 @@ I: _I_nfected and currently experiencing a diarrheal disease bout
 
 Data Descriptions
 -----------------
+	 
+	 
+.. list-table:: State definitions and initialization
+   :widths: 5 30
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - I
+     - Currently has diarrheal disease.
+   * - S
+     - Does not currently have diarrheal disease.
+	 
+.. csv-table:: Prevalence
+   :header: State,Prevalence,Source
+   :widths: 5, 10, 10
+   :stub-columns: 0
+
+   I,prev_302,como
```

Comment:
As we mentioned in the RT meeting, add a prefix to the id number to indicate if its a cause id, sequela id, rei id, or meid